### PR TITLE
Callbacks EN - tweak

### LINKED ISSUE #48: Callbacks: Delivery receipts clarification
### Callbacks

Perhaps we can make it more clear that all the values mentioned in `Callbacks > Message delivery receipts` are strings? It’s possible that someone skips to the table and doesn’t read the text preceding it (Not very high priority item)

### PR DESCRIPTION
**Pull request by anikbrazeau**
Attempt to fix #48 

> Perhaps we can make it more clear that all the values mentioned in Callbacks > Message delivery receipts are strings? It’s possible that someone skips to the table and doesn’t read the text preceding it (Not very high priority item)

### CODE REVIEW

File: src/en/callbacks.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This functionality works with test API keys, but does not work with smoke testin
 
 The callback message is formatted in JSON. All of the values are strings. The key, description and format of the callback message arguments will be:
 
-|Key | Description | Format|
+|Key string | Description | Format|
```
Comment (RenDavidDimen) [inline src/en/callbacks.md:37]: I believe for JSON, the Key value is always a string. In the same vein however, we could update the header to say `String format` to indicate that values associated with a key are strings 🙂

**Review by RenDavidDimen — CHANGES_REQUESTED** [review]
Thanks for taking a look into this! I just had a suggestion building on your changes that might help clarify the values in the table 🙂

File: src/en/callbacks.md (hunk 1 above)
Comment (anikbrazeau) [reply src/en/callbacks.md:37]: Oo thanks! will do!

**Review by RenDavidDimen — APPROVED** [review]
Looks good!

**Pull request merged**